src/screens/qr/components/BaseQRCodeScreen.tsx

```diff
@@ -17,15 +17,17 @@ export const BaseQRCodeScreen = ({children, showBackButton, showCloseButton}: Ba
   return (
     <Box backgroundColor="overlayBackground" flex={1}>
       <SafeAreaView style={styles.flex}>
-        <Box marginBottom="m">
-          <Toolbar
-            navText={i18n.translate('DataUpload.Close')}
-            showBackButton={showBackButton}
-            showCloseButton={showCloseButton}
-            useWhiteText={false}
-          />
-        </Box>
-        <View style={styles.flex}>{children}</View>
+        <ScrollView>
+          <Box marginBottom="m">
+            <Toolbar
```

Comment:
I think the Toolbar should be outside of the scrollview so the back and close buttons are always present at the top of the screen, similar to how the BaseDataSharingView works.